SlicerMONAIViz.py

```diff
@@ -438,7 +438,16 @@ def onAddTransform(self):
 
         t = self.ui.transformsComboBox.currentText
         m = self.ui.modulesComboBox.currentText
-        self.addTransform(-1, None, t, None)
+
+        v = ""
+        if t[-1] == "d":  # this is a dictionary transform
+            # now exclude some transforms whose name happens to end with d
```

Comment:
I think list transform has this flag to check if it's a dictionary transform

Reply:
https://github.com/Project-MONAI/SlicerMONAIViz/blob/main/SlicerMONAIVizLib/utils.py#L46

Reply:
Thank you for the suggestion. I just searched, and I could not find a `isDictionary()` or something similar.

I checked `Allow edits by maintainers`. Please feel free to amend the commit, or add a new commit which does this better.

Reply:
I now see https://github.com/Project-MONAI/SlicerMONAIViz/blob/main/SlicerMONAIVizLib/utils.py#L46. I am still unsure how to use it properly.

Reply:
thanks, Sachi is out of office today, I'm merging this for now and we can follow up on the dictionary types.

Reply:
I created #15 to track this.